iphone/cli/commands/_build.js

```diff
@@ -4048,7 +4055,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 
 					case 'png':
 					case 'jpg':
-						imageAssets[relPath] = info;
+						useAppThinning ? imageAssets[relPath] = info : resourcesToCopy[relPath] = info;
```

Comment:
This is hard to read. Just use a plain old if/else please. I have no problem doing inline if's, however you're assigning a l-value and it's easy for people to get confused.